internal/client.go

```diff
@@ -778,6 +778,10 @@ type (
 		// WARNING: Task queue priority is currently experimental.
 		Priority Priority
 
+		// resultInfo - Optional pointer to store information of StartWorkflowExecution response.
+		// Only settable by the SDK - e.g. [temporalnexus.workflowRunOperation].
+		resultInfo *startWorkflowResultInfo
```

Comment:
I would expect this field and the result info to be exported to be usable by anyone outside of this library

Reply:
This should not be exposed

Reply:
Oh, I didn't see it being used by Nexus, I though it was just a mistake that it wasn't exposed

Reply:
I'll have a separate PR for the Nexus change. I can do at once, just thought it would be easier to review as two PRs